rest/attachment_test.go

```diff
@@ -2276,7 +2275,7 @@ func TestUpdateViaBlipMigrateAttachment(t *testing.T) {
 		ds := rt.GetSingleDataStore()
 		ctx := base.TestCtx(t)
 
-		initialVersion := btc.rt.PutDoc(doc1ID, `{"_attachments": {"hello.txt": {"data": "aGVsbG8gd29ybGQ="}}}`)
+		initialVersion := btc.rt.PutDocDirectly(doc1ID, JsonToMap(t, `{"_attachments": {"hello.txt": {"data": "aGVsbG8gd29ybGQ="}}}`))
```

Comment:
What do you think of a function like:


```
// PutDocWithAttachment will upsert the document with a given contents and attachments.
func (rt *RestTester) PutDocWithAttachment(docID string, body string, attachmentName, attachmentBody string) DocVersion {
	// create new body with a 1.x style inline attachment body like `{"_attachments": {"camera.txt": {"data": "Q2Fub24gRU9TIDVEIE1hcmsgSVY="}}}`.
	require.NotEmpty(rt.TB(), attachmentName)
	require.NotEmpty(rt.TB(), attachmentBody)
	var rawBody db.Body
	require.NoError(rt.TB(), base.JSONUnmarshal([]byte(body), &rawBody))
	require.NotContains(rt.TB(), rawBody, db.BodyAttachments)
	rawBody[db.BodyAttachments] = map[string]any{
		attachmentName: map[string]any{
			"data": attachmentBody,
		},
	}
	// call PutDocDirectly since PutDoc will not populate CV yet (file a ticket!)
	return rt.PutDocDirectly(docID, rawBody)
}
```

Then we can decide later of how we might want to create an attachment body?